api/views/schools/branches_viewsets.py

```
@@ -64,7 +64,7 @@ class SchoolBranchViewSet(viewsets.ModelViewSet):
     # Default ordering: headquarters first, then by name
     ordering = ['-is_headquarters', 'name']
     lookup_field = 'uuid'
-    permission_classes = [IsStaffOrSuperuserOrReadOnly]
+    permission_classes = [IsStaffOrSuperuserOrReadOnly, IsAuthenticatedOrReadOnly]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)